e2e-tests/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "e2e-tests"
 
-publish = false
+# publish = false
```

Comment:
Why comment out? Either remove or keep there?

Reply:
That's some debugging remnant, thank you for noticing this, I'll remove